addon/index.js

```diff
@@ -116,15 +129,66 @@ function setupApp() {
 
 // update the our icon for devtools themes
 Prefs.observe('devtools.theme', pref => {
-  setActionButton(pref === 'dark');
+  setToggleButton(pref === 'dark');
 });
 
-setActionButton(Prefs.get('devtools.theme') === 'dark');
+const panel = Panel({ // eslint-disable-line new-cap
+  width: PANEL_WIDTH,
+  height: PANEL_HEIGHT,
+  contentURL: './feedback.html',
+  contentScriptFile: './panel.js',
+  onHide: () => {
+    button.state('window', {checked: false});
+  }
+});
+
+panel.on('show', showExperimentList);
+panel.port.on('back', showExperimentList);
+
+function showExperimentList() {
+  panel.port.emit('show', getExperimentList(store.availableExperiments,
+                                            store.installedAddons));
+}
+
+panel.port.on('link', url => {
+  require('sdk/tabs').open(url);
+  panel.hide();
+});
+
+panel.port.on('launch-feedback', id => {
+  panel.port.emit('show', getFeedbackForm(id));
+});
+
+panel.port.on('feedback-submit', dataStr => {
+  const data = JSON.parse(dataStr);
+  data.tags = ['main-addon'];
+  authRequest.sendMetric(settings, 'user-feedback', data,
```

Comment:
Also, we probably want to revisit this. Not sure we want to send feedback as a metric ping - we have [`/api/feedback`](https://github.com/mozilla/testpilot/blob/master/testpilot/experiments/views.py#L89) that we use with [the feedback form view](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/views/feedback-view.js#L81) on the main site. Should probably use the same thing to keep feedback responses together.